tajo-client/src/main/java/org/apache/tajo/client/QueryClient.java

```diff
@@ -94,11 +97,11 @@
    *
    * @return If failed, return null.
    */
-  ResultSet executeQueryAndGetResult(final String sql) throws SQLException;
+  ResultSet executeQueryAndGetResult(final String sql) throws TajoException;
```

Comment:
Take a look at the signature ``executeQuery`` and ``executeQueryWithJson``. They return SubmitQueryResponse which contains ReturnState. So, we should allow callers to handle the return state directly.